samples/attested_tls/server/enc/cert_verify_config.cpp

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+#include "cert_verify_config.h"
+
+oe_result_t verify_claim_value(const oe_claim_t* claim)
+{
+    oe_result_t result = OE_OK;
```

Comment:
What is the use of this function? It seems always return `OE_OK`.

Reply:
On the server side, we can't verify claim. During the build process, CLIENT_ENCLAVE_MRENCLAVE is not available as client image is built later. We can only verify the claim value of Server or Client, but not both.

Reply:
I will add same comments to the code.